src/ISEarth.c

```diff
@@ -25,6 +25,11 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 #define C_NEG_MG_DIV_KT0_F			-1.18558314779367E-04f		// - (M * g)  / (K * T0)
 #define C_NEG_KT0_DIV_MG_F			-8.43466779922578000E+03f	// - (K * T0) / (M * g)
 
+#ifndef C_ECEF_EPS
```

Comment:
Not a big deal, but it really has nothing to do with ECEF, more like for safe division and such. Maybe keep it generic, like IS_EPS or whatever?

Reply:
I renamed to C_ECEF_EPSILON to clarify the name.  I'm going to keep the ECEF in the name, since its only usage is in ECEF calculations.  If we end up using this in a broader context, where it justifying putting it back into ISConstants.h, then we can put it behind an IS namespace, and probably define it as a C++ constexpr, so it honors scope.  For now, this will work given its limited usage.